src/Gui/DlgSettingsNavigation.cpp

```diff
@@ -211,8 +211,9 @@ void DlgSettingsNavigation::loadSettings()
     // but not accessible if there is no cube yet drawn

```

Comment:
do you know why it isn't available? This is of course nothing that should stop this PR as it is a previous problem but duplicating the logic to calculate the font size means that we may forget to update one of them if we ever decide to change this.

Reply:
> do you know why it isn't available?

From where should the info come? There is no parameter yet written thus the info is not yet available.

> This is of course nothing that should stop this PR

OK, merging then.